server/models/SpotifyToken.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const SpotifyTokenSchema = new mongoose.Schema({
+  userId: String,
+  spotifyId: String,
+  accessToken: String,
+  refreshToken: String,
+  expiresIn: Number,
```

Comment:
A more standard way to represent this is to call it expiresAt and have it also be a date type.